Remove sass lint (#1902)
alphagov/govuk-frontend: docs/contributing/coding-standards/css.md

# Linting

To ensure code quality and consistency in our Sass files we check that certain

> style rules are followed. These rules are based on [stylelint-config-gds](https://github.com/alphagov/stylelint-config-gds/blob/main/scss.js), but we also add our own custom rules with a project [YAML file](../../../.stylelintrc.yml)

Reviewer: ```suggestion
style rules are followed. These rules are based on [stylelint-config-gds](https://github.com/alphagov/stylelint-config-gds/blob/main/scss.js), but we also add our own custom rules with a project [YAML file](../../../.stylelintrc.yml).
```

See [testing and linting](/docs/releasing/testing-and-linting.md) for more information.

## Running the lint task

> You can run the linter in gulp: `gulp scss:stylelint` or check linting directly in [Sublime Text](https://github.com/SublimeLinter/SublimeLinter-stylelint), [Atom](https://atom.io/packages/linter-stylelint) or other [IDE's](https://stylelint.io/user-guide/integrations/editor)

Reviewer: ```suggestion
You can run the linter in gulp by running `gulp scss:stylelint`, or use linting in [Sublime Text](https://github.com/SublimeLinter/SublimeLinter-stylelint), [Atom](https://atom.io/packages/linter-stylelint) or [other editors that support stylelint](https://stylelint.io/user-guide/integrations/editor).
```

$my-example-var: value;
```

> ### Don't write trailing zeros for numeric values with a decimal point

Reviewer: ```suggestion
### Don't write trailing zeroes for numeric values with a decimal point
```

Good:
```
.selector {
  font-size: .5em;
}
```

### Remove trailing whitespace

> More write up on [supported rules](https://github.com/stylelint/stylelint/blob/master/docs/user-guide/configure.md#rules).

Reviewer 2: I'm not entirely sure what the need for this link is, but wondering if https://stylelint.io/user-guide/rules/about or https://stylelint.io/user-guide/rules/list might be a better place to link to – the current URL seems to be more about configuring rules or adding entirely new ones.
